src/Network/Network/PrivateLinkService/NewAzurePrivateLinkServiceCommand.cs

```diff
@@ -117,6 +122,15 @@ private PSPrivateLinkService CreatePSPrivateLinkService()
                 psPrivateLinkService.AutoApproval.Subscriptions = AutoApproval.ToList();
             }
 
+            if (this.EnableProxyProtocol.IsPresent)
```

Comment:
`psPrivateLinkService.EnableProxyProtocol = this.EnableProxyProtocol.IsPresent` would be cleaner